tests/unit/test_store.py

```diff
@@ -130,7 +130,7 @@ def test_store_can_create_s3_fsspec_session(self):
 
 
 @pytest.mark.xfail(
-    reason="This test reproduces a bug (#610) which has not yet been fixed."
+    reason="Expected failure: Reproduces a bug (#610) that has not yet been fixed."
```

Comment:
Not sure how I feel about including the status in the reason string, but don't feel strongly enough to request anything. If it's helping someone, it's helping someone!

Reply:
I did this because `xfail`, `xfailed` and `XFAIL` require you to know Pytest well enough to interpret them as "expected failure".  As soon as you *know* that, the x = expected makes sense, but if you don't there's nothing really to help you get there. 

I think it's a little more general contributor-friendly this way, but I'm fine leaving this as a "Google it" thing if you don't understand.

Reply:
That's a good enough reason for me ;)